### DISCUSSION TITLE
Build is pending!

### DISCUSSION THREAD

**Discussion by shapathdas** [AntlerVC/firetable#1460] [Support / Q&A] [answered]
**Describe the bug**
When I am deploying a cloud function. Building is not happening. After the click it is displaying "Build pending" and the build is not progressing.

**Comment by shapathdas** [comment]
<img width="749" alt="Screenshot 2023-10-01 at 11 19 37 PM" src="https://github.com/rowyio/rowy/assets/112649795/d808be5b-3960-42eb-a873-e0fdfb730858">

**Comment by harinij** [comment, marked as answer]
@shapathdas are you building a derivative or webhook/extension? Sometimes cloud functions need a second trigger to trigger the build and can be tricky depending on your specific function. We actually recently launched [buildship.com](https://buildship.com/) to address this exact situation and give an instant deploy experience without long build of cloud functions and this is compatible with Rowy. Here is the buildship [documentation](https://docs.buildship.com/tutorials/firebase-project) on it in case you want to give this a try.
